### DISCUSSION TITLE
Can't run .exe file | Windows blocks

### DISCUSSION THREAD

**Discussion by Miguelitoxxx** [kubilus1/autoortho#255] [Q&A] [answered]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

-  Windows 11
-XP 11
- AutoOrtho Version - The last one 
- Filesystem type (NTFS, exFAT, etc) (idk what is this)

### question

Basically, I download the autoortho_win zip file and I tried to run it, but my windows it's blocking it. I tried to disable all anti-virus systems and still, the same error.

This is a new PC with windows 11, and the old one it worked just fine but it was on windows 10. I don't know what can I do now as it's the first time it gives me this error.

**To Reproduce**
I just try to run this file. 
![image](https://github.com/kubilus1/autoortho/assets/92696605/e2ac96d9-5d1a-48ff-8b0a-c74f1ba80273)

**Screenshots**
![image](https://github.com/kubilus1/autoortho/assets/92696605/20698a7d-fd8b-473f-a6f8-72cef64b00df)

**Comment by dpdurst** [comment, marked as answer]
What folder are you trying to run this from?

**Reply by Miguelitoxxx** [reply]
Thanks man :D your question helped me a lot, you have no idea AHHAHAH <3
